src/FINDOLOGIC/Export/Data/Item.php

```diff
@@ -257,7 +257,7 @@ public function setBonus(Bonus $bonus)
      * @param float|string $bonus The bonus value of the item.
      * @param string $usergroup The usergroup of the bonus value.
      */
-    public function addBonus($bonus, $usergroup = '')
+    public function addBonus($bonus, string $usergroup = '')
```

Comment:
Type for `$bonus` is missing. Since this will be in a new major release, we can break compatibility and decide to only allow `float`.

Reply:
Fixed